### PR TITLE
Fix order of message replacing patterns

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Comment by temanbrcom** [comment]
Requires also the tests for the new tokens.

**Comment by temanbrcom** [comment]
@schte01

**Review by asatklichov — APPROVED** [review]
🏞️

File: server/src/main/java/org/eclipse/lsp/cobol/core/strategy/CobolErrorStrategy.java [hunk 1]
```diff
@@ -43,28 +46,35 @@
   private static final String REPORT_UNWANTED_TOKEN = "reportUnwantedToken";
   private static final String REPORT_MISSING_TOKEN = "reportMissingToken";
   private static final String SPECIAL_TOKEN_HANDLING_FILEPATH = "SpecialTokenHandling.properties";
-  private static final LinkedHashMap<String, String> SPECIAL_TOKEN_HANDLING_MAP;
+  private static Map<String, String> specialTokenHandlingMap;
```
Comment (grianbrcom) [inline server/src/main/java/org/eclipse/lsp/cobol/core/strategy/CobolErrorStrategy.java:49]: Why it's not a final? Nothing was changed from that point of view.

**Review by grianbrcom — APPROVED** [review]
:frog:

File: server/src/main/java/org/eclipse/lsp/cobol/core/strategy/CobolErrorStrategy.java (hunk 1 above)
Comment (temanbrcom) [reply server/src/main/java/org/eclipse/lsp/cobol/core/strategy/CobolErrorStrategy.java:49]: When it is final, it produces some error about initialization

**Pull request merged**